src/helpers/sql.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+use Illuminate\Database\Query\Builder;
+if (!function_exists('dumpSql')) {
```

Comment:
This check is not needed because it is done in the service provider. These should also be a blank line between the import and the function.